docs/source/user_guide/troubleshooting.rst

```diff
@@ -0,0 +1,60 @@
+.. _user_guide_troubleshooting:
+
+===============
+Troubleshooting
+===============
+In this page, users can find ideas to resolve the most common issues encountered with pydpf-core.
+Ways to improve users codes will also be exposed.
+
+
+Using the Server
+----------------
+
+Starting DPF Server
+~~~~~~~~~~~~~~~~~~~
+While using the dpf python API to start the server with :py:meth:`start_local_server()
+<ansys.dpf.core.server.start_local_server>` or while starting the server manually (with ''Ans.Dpf.Grpc.sh''
```

Comment:
Use ` instead of ' to get it rendered properly within Sphinx.

Reply:
@akaszynski I'm not sure to understand what you mean here sorry